### DISCUSSION TITLE
Voice volume

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#207] [Q&A] [answered]
Cruiser just keeps getting better.
I was on the road for 2 weeks (France, Pyrinées) -> on the whole everything works fine.
One problem, however: the volume!
I was traveling with the convertible. From 80km/h you can hardly understand anything.
The speed camera warning is significantly louder

**Comment by devemux86** [comment]
@Leeb64 thanks for the kind words!

Voice volume in Cruiser can be adjusted in "Settings | Voice guidance | Voice volume".

The default "System" option uses the device's volume for the selected audio output channel.
Or you can change it to a custom voice volume, e.g. 100%, have you tried that?

**Comment by devemux86** [comment]
Also Google Text-to-Speech has extra volume options in the Android language settings.

> The speed camera warning is significantly louder

Is it speech or some other sound?

**Comment by devemux86** [comment]
I am also interested in what all users have to say about the voice volume in Cruiser navigation.

Do you all have something to report, what to change something?

**Comment by Leeb64** [comment]
The smartphone with the low Volume is a Sony Experia2.
All settings to max.
My other smartphone, a Doogee s58 is louder.
My wifes smartphone,a Redmi Note 10 5G,is also louder.
Is there a way to increase the volume?(ca 50%)?

**Comment by Leeb64** [comment]
I found a App : Super Volume.
I can increase the volume with the app.
But the whole thing is cumbersome. First open the Super Volume app, then Cruiser...
Plus everything else gets louder too.
Not optimal.
and then this annoying ad....

**Comment by devemux86** [comment]
@Leeb64 I will check what else is possible.

Have you changed the "Settings | Voice guidance | Voice volume" in Cruiser, like 100%?

Also Google TTS Options has an "Amplify speech volume" setting:

<img src="https://github.com/devemux86/cruiser/assets/3484020/627ae924-7eed-472e-9bdb-4542caff13c8" width="33%"/>

**Comment by airald** [comment]
I use system volume and automate set up system volume at 100% when the phone connects with the sena headset that I use when I ride my motorcycle or Parrot if I drive my car. Then I change volume when needed (both Sena and Parrot changes volume automatically depending on ambient sound level).

**Comment by devemux86** [comment]
Next Cruiser will have improvements in voice navigation:
- "**Voice volume**": will work better

- "**Prepare audio connection**": better support for all "**Voice guidance output**" channels
(no cut words!)

Please test the voice navigation with the next Cruiser. :slightly_smiling_face:

**Reply by devemux86** [reply, marked as answer]
> Next Cruiser will have improvements in voice navigation

Available in [Cruiser 3.0.23](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by airald** [reply]
tested it! no more cut words. Not really a problem, but now is better, of course.

**Reply by devemux86** [reply]
@airald thanks for the feedback!

> tested it! no more cut words.

What output do you use in Cruiser?

(Media / music or Notification or Phone call)

**Reply by airald** [reply]
I use Media/music.
